app/views/components/_figure.html.erb

```diff
@@ -4,18 +4,19 @@
   credit ||= ''
 %>
 <figure class="app-c-figure" lang="en">
-  <img class="app-c-figure__image" src="<%= src %>" alt="<%= alt %>">
-    <figcaption class="app-c-figure__figcaption">
-      <% unless caption.blank? %>
-        <p class="app-c-figure__figcaption-text">
-          <%= caption %>
+  <% unless src.blank? %>
```

Comment:
It's a really minor nitpick, but I know our linter prefers `if src.present?` over `unless src.blank?`.  We don't lint our views, but it's probably good practice to do that here too.

Reply:
On the same point, I know you didn't write that code, but `unless caption.blank?` below should also be changed to `if caption.present?`.

Reply:
Good shout. Updated.